p2p/src/node.rs

```diff
@@ -1639,6 +1639,7 @@ mod tests {
     }
 
     #[test(tokio::test)]
+    #[ignore]
```

Comment:
should we keep this? it fails almost every time on osx but seems reliable on linux, haven't sorted out the issue yet.